docs/DeveloperGuide.md

```diff
@@ -308,18 +313,72 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
+**Use case: Update Contact Information**
+
+**MSS**
+
+1.  User requests to list TAs
+2.  AddressBook shows a list of TAs
+3.  User requests to update the contact of a specific TA in the list and key in the necessary information
```

Comment:
Can be broken into two steps, so that if information keyed in is invalid, the user doesn't have to select the TA again

Reply:
for this one I am following the current implementation of AB3, breaking into two steps require code changes so we will have to discuss if we want to go ahead with that